ciao-launcher/vmconfig.go

```diff
@@ -96,6 +96,14 @@ func (cfg *vmConfig) findVolume(UUID string) *volumeConfig {
 	}
 	return nil
 }
+func (cfg *vmConfig) haveBootableVolume() bool {
```

Comment:
The bootable volume must be the first volume in the volume list.  We cannot really sort the volumes to ensure this as doing so would change the names that the volumes are given in the guest.  Maybe there's some nice way of assigning names in volumes going forward, but this is currently the way things are.  So this function should be

```
func (cfg *vmConfig) haveBootableVolume() bool {
  return len(cfg.Volumes) > 0 && cfg.Volumes[0].Bootable
}
```